week5/1-virtuoso-hybrid-store.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+---
+
+# Virtuoso, a Hybrid RDBMS/Graph Column Store
+
+___
+
+This paper discusses the design choices met in applying column store techniques under the twin requirements of performing well on the unpredictable, semi-structured RDF data and more typical relational BI workloads. Virtuoso started out as a row-wise transaction oriented RDBMS and transitioned to being an RDF graph store and subsequently started using column-wise storage and vectored execution. The main incentive for using column-wise storage was its excellent space efficiency, which suited Virtuoso's large RDF applications.
```

Comment:
`RDF` I don't know what this means. Consider adding a full form first before using its acronym.